source/contributing/writing-documentation.rst

```diff
@@ -4,31 +4,41 @@
 Writing documentation
 #####################
 
-This section gives some guidelines to write consistent documentation
+This section provides some guidelines for writing consistent documentation
 for |project_name|.
 
 
 
 .. seealso::
 
   The quickest and easiest way to contribute to the documentation is to sign up
-  for a `GitHub account <https://github.com/>`_ and edit the documentation pages
-  (See :repo:`doc`).
+  for a `GitHub account <https://github.com/>`_ and edit the documentation pages by clicking the 
+  "Edit on GitHub" link at the top of the page. (See :repo:`doc`).
 
 
-Build the docs
-==============
+Building the docs
+=================
 
+The following tools are required to build the documentation from source:
```

Comment:
Maybe we should get all the information from https://github.com/geonetwork/doc/tree/develop#project-documentation-module here and then point to the doc so we don't have 2 places to maintain. What do you think ?

Reply:
I didn't even know about that page :-) yes definitely